app/components/post-search-results-item.js

```diff
@@ -0,0 +1,25 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+import Thumbnailer from '../modules/thumbnailer';
+
+export default Component.extend({
+  isQuizWithTakes: computed('post.{type,stats,stats.takes}', function () {
```

Comment:
I think you can just use `post` as a dependency for this computed property. I assume that `post` is not mutated.

Reply:
Fixed in 1546f68